zuul.d/projects.yaml

```diff
@@ -50,6 +37,13 @@
             branches: master
         - spark-integration-test-minikube-k8s:
             branches: master
+            
+- project:
+    name: kubernetes-sigs/kind
+    periodic:
```

Comment:
Hi @mrhillsman , we have done refactor for periodic pipeline, you add `kind` job at here in block `periodic-0/12`, but refer pipeline `periodic`, it's obvious error, please check document and choose a suitable pipeline. 

https://docs.openlabtesting.org/publications/contributors/development-guide#periodic-job-and-pipeline-schedule